aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,183 @@
+import { Example } from '../../layouts';
+import { 
+  PageBackGroundExample,
+  PageExample, 
+  PageFullExample,
+  PageNarrowExample,
+  SinglePageContent,
+  PageWideExample 
+} from '../../examples';
+
+Page makes it easy to implement [page layouts](/templates/page-layouts) uniformly across HPE 
+applications. Two components, Page and PageContent, work together to ensure HPE theme-defined 
+padding, horizontal alignment, max-width, and responsive behaviors are applied in a consistent, 
+turn-key manner.
+
+<Example 
+  docs="https://v2.grommet.io/page?theme=hpe"
+  showResponsiveControls={['fullScreen']}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  screenContainer
+  width="100%"
+>
+  <PageExample />
+</Example>
+
+## Page and PageContent Working Together
+Page is the outer layout component and provides the desired layout context. 
+Page’s `kind` determines which theme-defined alignment, padding, and min/max-widths 
+are applied to its PageContent children.
+
+As a child of Page, PageContent consumes and adheres to the 
+layout styling provided by its parent Page.
+
+```
+<Page kind="wide">
+  <PageContent>
+
+    /* All contents within me will respect the 
+     * max-width, alignment, and padding specified 
+     * by Page's context. */
+     
+  </PageContent>
+  
+  /* Contents outside of PageContent are not bound by 
+   * Page's context. */
+
+</Page>
+```
+
+<br />
+
+In most cases, all content on a page can be wrapped in a [single PageContent component](#simple). 
+However, designs using [differing background colors](#full-width-background-color) to delineate page 
+sections may accomplish this by applying a PageContent per desired section.
+
+## Example use cases
+
+### Simple
```

Comment:
To make the story flow smoother, how about we keep it simple and say:
"Example: Content within a single PageContent"

"Simple" throws me off a little bit because I start viewing it as a variant

Reply:
I would agree here the `simple` makes me think this should be under the variants

Reply:
done